team-fingerprints-web/src/components/Breadcrumbs/__tests__/Breadcrumbs.spec.tsx

```diff
@@ -0,0 +1,101 @@
+import { screen } from "@testing-library/react";
+import Breadcrumbs from "../Breadcrumbs";
+import { renderWithRouter } from "test/testHelpers";
+import userEvent from "@testing-library/user-event";
+
+const mockedUsedNavigate = jest.fn();
+
+jest.mock("react-router-dom", () => ({
+  ...(jest.requireActual("react-router-dom") as any),
+  useNavigate: () => mockedUsedNavigate,
+}));
+
+const companyManagement = "/companies/623055978d7217f258110b7e";
+const teamManagement = "/team/623c9a2b4e55154cf4f8d6e7";
+const surveyResults = "/surveys/623b12ef98807f8836773758";
+
+describe("Breadcrumbs", () => {
+  describe("rendering correct content on a specified page url", () => {
+    it("renders first bookmark only", () => {
+      renderWithRouter(<Breadcrumbs />, {
+        route: companyManagement,
+      });
+
+      const companyManagementEl = screen.getByText("Company Management");
+
+      expect(companyManagementEl).toBeInTheDocument();
+    });
+
+    it("renders first two bookmarks when path is default", () => {
+      renderWithRouter(<Breadcrumbs />, {
+        route: `${companyManagement}${teamManagement}`,
+      });
+
+      const companyManagementEl = screen.getByText("Company Management");
+      const textManagementEl = screen.getByText("Team Management");
+
+      expect(companyManagementEl).toBeInTheDocument();
+      expect(textManagementEl).toBeInTheDocument();
+    });
+
+    it("renders first three bookmarks", () => {
+      renderWithRouter(<Breadcrumbs />, {
+        route: `${companyManagement}${teamManagement}${surveyResults}`,
+      });
+
+      const companyManagementEl = screen.getByText("Company Management");
+      const teamManagementEl = screen.getByText("Team Management");
+      const surveyListEl = screen.getByText("Survey List");
+
+      expect(companyManagementEl).toBeInTheDocument();
+      expect(teamManagementEl).toBeInTheDocument();
+      expect(surveyListEl).toBeInTheDocument();
+    });
+
+    it("renders all bookmarks", () => {
+      renderWithRouter(<Breadcrumbs />, {
+        route: `${companyManagement}${teamManagement}${surveyResults}`,
+      });
+
+      const companyManagementEl = screen.getByText("Company Management");
+      const teamManagementEl = screen.getByText("Team Management");
+      const surveyListEl = screen.getByText("Survey List");
+      const surveyResultsEl = screen.getByText("Survey Results");
+
+      expect(companyManagementEl).toBeInTheDocument();
+      expect(teamManagementEl).toBeInTheDocument();
+      expect(surveyListEl).toBeInTheDocument();
+      expect(surveyResultsEl).toBeInTheDocument();
+    });
+  });
+
+  describe("triggering navigate based on isActive variable", () => {
+    it("does not navigate", () => {
+      renderWithRouter(<Breadcrumbs />, {
+        route: `${companyManagement}${surveyResults}`,
+      });
+
+      const surveyResultsEl = screen.getByText("Survey Results");
+
+      userEvent.click(surveyResultsEl);

```

Comment:
You should use `await` for click user event
```suggestion
      await userEvent.click(surveyResultsEl);
```

Reply:
Done